src/components/forms/FormControls.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormControl, ButtonGroup, DropdownButton, MenuItem, Checkbox } from 'patternfly-react';
+import './FormControls.css';
+
+export const TextAreaControl = ({ fieldKey, value, onChange }) => (
+  <FormControl componentClass="textarea" value={value} onChange={event => onChange(event.target.value, fieldKey)} />
+);
+
+TextAreaControl.propTypes = {
+  fieldKey: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired
+};
+
+export const DropdownControl = ({ fieldKey, value, choices, onChange }) => (
```

Comment:
I personally prefer a component in a file, it makes it easier to find, maybe just change them to forms/textArea, forms/checkBox etc, would be trivial to find ?